specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "App Compliance Automation Tool for Microsoft 365",
+    "version": "2022-11-16-preview",
+    "description": "App Compliance Automation Tool for Microsoft 365 API spec"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.AppComplianceAutomation/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "AppComplianceAutomation"
+        ],
+        "description": "Query the ongoing operation status.",
+        "operationId": "OperationResults_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
```

Comment:
Dont you need an actual response body here for the operation result itself ?

Reply:
@raosuhas, This is a historical API, at the very begginning, we were plan to add an API to return the operation status with a body. But finally we turn to using the RPaaS. So that not support this API. Maybe we can think about how to mitigate it. Maybe we can remove it.

Reply:
Hi @raosuhas ~
This historical API has been removed. See [here](https://github.com/Azure/azure-rest-api-specs/pull/21152/files#diff-16cc0066db31d47ee8b4f0e9385ece034881677d74a14b4d48790444b24bda4d).